Framework/Geometry/inc/MantidGeometry/Instrument.h

```diff
@@ -212,6 +212,7 @@ class MANTID_GEOMETRY_DLL Instrument : public CompAssembly {
   // Methods for use with indirect geometry instruments,
   // where the physical instrument differs from the 'neutronic' one
   boost::shared_ptr<const Instrument> getPhysicalInstrument() const;
+  bool hasPhysicalInstrument() const;
```

Comment:
Why extend the interface of legacy code? Isn't `if(getPhysicalInstrument()` just as good?

Reply:
Yes, it's just as good. It is simply a readability aid.